packages/editor-ui/src/views/SettingsCommunityNodesView.vue

```diff
@@ -0,0 +1,134 @@
+<template>
+	<SettingsView>
```

Comment:
there's a bit of jerk in content change when switching from users to community nodes page.. it's very annoying. can you please fix this. Maybe don't render page until packages are done fetching.. feel free to reach out if it's not clear to you what the issue here is

Reply:
Just tested. This jerking happens when there are no packages installer. In that case the endpoint responds almost instantly so loader appears and immediately disappears. Will delay loader hiding a bit if there are no packages.